src/cmds/jtag.c

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file
+ * @brief Enable JTAG pins
+ *
+ * @date 27.06.2021
+ * @author kpishere
+ */
+
+#include <unistd.h>
+#include <stdio.h>
+#include <drivers/gpio/gpio.h>
+#include <drivers/gpio/bcm283x/bcm283x_gpio.h>
+
+int main(int argc, char **argv) {
+
+	bcm283x_gpio_jtag();
```

Comment:
It seems as bcm283x specific cmd. So May be place it in platform/raspbery folder?

Reply:
Oh, I see,  OK.  saw some examples in other platform targets.  great.